openairclim/calc_response.py

```diff
@@ -131,3 +141,31 @@ def calc_resp_all(config, resp_dict, inv_dict):
             out_inv_dict[year] = out_arr
         out_dict[spec] = out_inv_dict
     return out_dict
+
+
+def calc_resp_sub(config, species_sub):
+    """
+    Calculates responses for specified sub-species.
```

Comment:
I would add some more description explaining that the results for the main species must be available and calculated before the results for the sub-species can be calculated (that's what I understand this function requires).

Reply:
This make sense. description added.